src/azure-cli/requirements.py3.Darwin.txt

```diff
@@ -132,7 +132,7 @@ scp==0.13.2
 semver==2.13.0
 six==1.14.0
 sshtunnel==0.1.5
-tabulate==0.8.3
+tabulate==0.8.9
```

Comment:
Old `tabulate` uses string to compare version, resulting in `"10" < "3"`, which is certainly wrong.

```
    if python_version_tuple() >= ("3", "3", "0"):
        from collections.abc import Iterable
    else:
>       from collections import Iterable
E       ImportError: cannot import name 'Iterable' from 'collections' (/opt/hostedtoolcache/Python/3.10.0/x64/lib/python3.10/collections/__init__.py)

env/lib/python3.10/site-packages/tabulate.py:16: ImportError
```